packages/eslint-config-udemy-jasmine-addons/tests.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// noop
+const eslintConfigTester = require('eslint-config-tester');
+const path = require('path');
+
+eslintConfigTester.runOnText(
```

Comment:
@cansin just to restate what we talked offline for the record, I suggest having a top-level comment in this file saying that the sole purpose of running `runOnText` is to validate the eslint config rather than testing if the rules work. So these files shouldn't be modified to change the example text you have below.